src/import/k8s.ts

```diff
@@ -64,6 +64,19 @@ export class ImportKubernetesApi extends ImportBase {
     const typeGenerator = new TypeGenerator({
       definitions: schema.definitions,
       exclude: this.options.exclude,
+      renderTypeName: (def: string) => {
+        const type = def.split('.');
+
+        if (type.length < 2) {
+          // this can happen with synthetic definitions we inject.
+          // e.g KubeMutatingWebhookConfigurationV1Beta1Props
+          return def;
+        }
+
+        const basename = type[type.length - 1];
+        const version = type[type.length - 2];
+        return getTypeName(basename, version);
```

Comment:
So right now just implementing the same postfix logic as our constructs. We still need to figure out what to do with all the rest of the namespace. But we can do that in a subsequent PR